### PR TITLE
Add wrap files for brotli-1.0.7

### PR DESCRIPTION
**Pull request by dcbaker**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  ],
+  c_args : c_args,
+  include_directories : inc,
+  link_with : libbrotli_common,
+  dependencies : dep_m,
+  version : meson.project_version(),
+  install : install,
+)
+
+dep_brotli_encoder = declare_dependency(
```
Comment (legeana) [inline meson.build]: The common style is to use `${package}_dep`.
Here and in other places.

File: meson.build [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+project(
+  'brotli',
```
Comment (legeana) [inline meson.build]: `google-brotli`

**Review by legeana — CHANGES_REQUESTED** [review]

**Comment by dcbaker** [comment]
@legeana, look good now?

**Comment by legeana** [comment]
@dcbaker looks good, thank you for updating it :)

**Pull request merged**